workspaces/ui-v2/src/lib/diff-description-interpreter.ts

```diff
@@ -8,14 +9,53 @@ import {
 import { getExpectationsForShapeTrail } from './shape-diff-dsl-rust';
 import { code, ICopy, plain } from '<src>/pages/diffs/components/ICopyRender';
 import { IJsonObjectKey } from '@useoptic/cli-shared/build/diffs/json-trail';
-//@ts-ignore
-const { toJsonExample } = require('@useoptic/shape-hash');
+import { IHttpInteraction } from '@useoptic/optic-domain';
+import { toJsonExample } from '@useoptic/shape-hash';
+
+const getJsonBodyToPreview = (
```

Comment:
Copied from below - the only difference it was pulling `location` in via a closure, now we pass it in explicitly so we don't need to duplicate